Inc/Util/util.h

```diff
@@ -8,12 +8,7 @@
 #ifndef INC_UTIL_UTIL_H_
 #define INC_UTIL_UTIL_H_
 
-//#include "cmsis_os.h"
-
-/* as we don't include cmsis_os.h, we need to define some datatypes ourselves */
-typedef u_int8_t uint8_t;
-typedef u_int16_t uint16_t;
-typedef u_int32_t uint32_t;
+#include "cmsis_os.h"
```

Comment:
cmsis_os is also a freeRTOS library, not needed for recovery

Reply:
Yes, I put everything related to freeRTOS into the `util.h`. Recovery will not need to include this file, as there are not any dependencies in the code